src/NuGet.Services.ServiceBus/TopicClientWrapper.cs

```diff
@@ -38,6 +38,28 @@ public TopicClientWrapper(string clientId, string clientSecret, string tenantId,
         }
 
         public Task SendAsync(IBrokeredMessage message)
+        {
+            BrokeredMessage innerMessage = GetBrokeredMessage(message);
```

Comment:
Nit: NuGet style is to use `var` wherever possible:

```suggestion
            var innerMessage = GetBrokeredMessage(message);
```